docs/ops/condition/If_8.md

```diff
@@ -0,0 +1,226 @@
+Also the number outputs from *If* always must be greater than zero and equal to the number outputs from each subgraph.
+
+**If attributes**:
+
+* **Subgraphs**:
+
+    `then_body`/`else_body` are subgraphs that will be executed depending on the `cond` value. 
+    The subgraph is described operation by operation as a typical IR network. 
+    The subgraph has parameters (`Parameter` operations) and results (`Result` operations).
+    
+    * **Subgraph's parameters** - inputs to the subgraph which associated with *If* inputs via *port_map*. 
+    The number of parameters for the subgraph can be any (even zero).
+    
+    * **Subgraph's results** - outputs from the subgraph which associated with *If* outputs via *port_map*.
+    The subgraph must contain at least one result. Each *If* output is associated with one result from the subgraph. 
+    It follows that number of `then_body` results is the equal to the number of outputs from the *If* and 
+    the number of `else_body` results. 
+    Type of the subgraph result and type of the associated output from *If* must be equal.
+    
+
+* **Port maps**:
+    
+    *port_map* is a set of rules to map input or output data tensors of *If* operation onto the subgraph data tensors. 
+    The `port_map` entries can be `input` and `output`. Each entry describes a corresponding mapping rule. 
+    *If* has two *port_maps* - `then_port_map` for `then_body` and `else_port_map` for `else_body`.
+
+    * **Port map attributes**:
+
+        * *external_port_id*
+            * **Description**: *external_port_id* is a port ID of *If* operation.
+            * **Range of values**: IDs of the *If* inputs and outputs
+            * **Type**: `unsigned int`
+            * **Default value**: None
+            * **Required**: *yes*
+
+        * *internal_layer_id*
+
+            * **Description**: *internal_layer_id* is a `Parameter` or `Result` operation ID inside 
+            the subgraph to map to.
+            * **Range of values**: IDs of the `Parameter` or `Result` operations in the subgraph
+            * **Type**: `unsigned int`
+            * **Default value**: None
+            * **Required**: *yes*
+
+**If Inputs**
+
+
+* **cond**: A scalar or 1D tensor with 1 element of `boolean` type specifying which an subgraph to execute. 
+`True` value means to execute the `then_body`, `False` - `else_body`. *Required*.
+
+* **Multiple other inputs**: tensors of different types and shapes. *Optional*.
+
+**If Outputs**
+
+* **Multiple outputs**: Results of execution of one of subgraph. Tensors of any type and shape.
+
+
+**Body Inputs**
+
+* **Multiple inputs**: tensors of different types and shapes. *Optional*.
+
+
+**Body Outputs**
+
+* **Multiple outputs**: Results of execution of the subgraph. Tensors of any type and shape.
+
+
+**Examples**
+
+*Example 1: a typical If structure*
+```xml
+    <layer id="6" name="PartitionedCall/model/if/cond" type="If" version="opset7">
```

Comment:
nit: not necessarily to have TensorFlow specific names for layers. Please call them using simple names.